pkgs/applications/editors/neovim/wrapper.nix

```diff
@@ -13,6 +13,9 @@
 neovim-unwrapped:
 
 let
+  # inherit interpreter from neovim
+  lua = neovim-unwrapped.lua;
```

Comment:
could we get this from callPackage above if it doesnt need to match neovim's version and only the utils are needed? It would reduce error

Reply:
it does need to match the neovim interpreter. the default LUA_PATH differs between lua versions. And when loading shared libraries, ABI differs as well

Reply:
Ah. Ok.